objectModel/Python/setup.py

```diff
@@ -51,14 +51,51 @@ def list_files_in_folder(folders):
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-
+dependencies = [
```

Comment:
We should read from the `requirements.txt` file instead of hard coding the values here.
See example of `long_description`.

Reply:
Can I break out the test deps from the requirements file?

I was particularly going it this way because some of the the hard test pins like mypy are giving my env issues.